packages/override-tools/src/Cli.ts

```diff
@@ -73,11 +76,24 @@ doMain(() => {
 });
 
 /**
- * Check that the manifest correctly describes files and is up to date
+ * Check that the given manifests correctly describe overrides and that all
+ * overrides are up to date
  */
+async function validateManifests(
+  manifestPaths: Array<string | number>,
```

Comment:
>| number [](start = 30, length = 8)

why is number even allowed?

Reply:
The typings for yargs say that an array from a variadic option can have numbers or strings. Guessing there is some logic where it will try to auto parse arguments as ints if integral.